CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.4.1 (October 17, 2020)
+
+- Better error logging when fetching AWS Secrets.
+- Updated httpOptions to use timeout instead of connectTimeout.
+- Use args interface for loadSecrets function.
```

Comment:
This is internal only so not really relevant to users of the library. I'd just remove this bullet point.